src/executors/default.yml

```diff
@@ -1,11 +1,11 @@
 description: >
   This is a sample executor using Docker and Node.
 docker:
-  - image: 'cimg/node:<<parameters.tag>>'
+  - image: 'cimg/python:<<parameters.tag>>'
 parameters:
   tag:
     default: lts
     description: >
-      Pick a specific circleci/node image variant:
-      https://hub.docker.com/r/cimg/node/tags
+      Pick a specific circleci/python image variant:
+      https://hub.docker.com/r/cimg/python/tags
```

Comment:
We don't have an executor, so this change seems redundant. I feel like this directory should maybe just be deleted.

Reply:
Yup, I know, since this orb is currently Python, this makes sense anyway (if we ever do add jobs).